oci/pull.bzl

```diff
@@ -131,6 +131,28 @@ When using a credential helper, it may be desirable to allow the built-in creden
 ```
 common --repo_env=OCI_GET_TOKEN_ALLOW_FAIL=1
 ```
+
+### Providing WWW-Authenticate challenges
```

Comment:
nit: the header should probably be about authenticating to registries, the tech detail of using www-authenticate header/challenge is probably not known to a casual reader

Reply:
i agree, in ideal case, we'd be able to read rctx.download  headers but since that's not possible, we'd have to expose this to advanced users.